packages/superset-ui-core/src/components/SafeMarkdown.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Moved from apache/superset, will delete from that codebase when I bump the superset-ui-core version used in superset